wp-includes/sqlite-ast/class-wp-sqlite-driver.php

```diff
@@ -2115,7 +2121,21 @@ private function translate( $node ): ?string {
 			case 'identifierKeyword':
 				return '`' . $this->translate( $node->get_first_child() ) . '`';
 			case 'pureIdentifier':
-				return $this->translate_pure_identifier( $node );
+				$value = $this->translate_pure_identifier( $node );
+
+				/*
+				 * At the moment, we only support ASCII bytes in all identifiers.
+				 * This is because SQLite doesn't support case-insensitive Unicode
```

Comment:
It's great to surface this error in an explicit way, thank you!

Thinking about the future, if identifiers are case-insensitive in PHP, could we lowercase them effectively in PHP and pass the transformed names to SQLite? Although that opens more questions – are column names raw bytes, or are they normalized? Yeah, perhaps not going there is for the best.